### PR TITLE
The "types" here seems redundant

### PR DESCRIPTION
**Pull request by lostsquirrel**
the types is hard to understand and according to the following content at line 535, so I think the "types" should not be existed.

### CODE REVIEW

File: content/en/docs/concepts/policy/resource-quotas.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ In release 1.8, quota support for local ephemeral storage is added as an alpha f
 ## Object Count Quota
 
 You can set quota for the total number of certain resources of all standard,
-namespaced resource types using the following syntax:
```
Comment (Aut0R3V) [inline content/en/docs/concepts/policy/resource-quotas.md:130]: I think this change might not be required. Can you specifically add a link for the document you're referring to?

Comment (lostsquirrel) [reply content/en/docs/concepts/policy/resource-quotas.md:130]: line 535 in this document

Comment (tengqm) [reply content/en/docs/concepts/policy/resource-quotas.md:130]: I don't think we need this change either.

Comment (bradtopol) [reply content/en/docs/concepts/policy/resource-quotas.md:130]: I don't believe this change is needed either

**Comment by bradtopol** [comment]
Hi @lostsquirrel thank you for your contribution.  I don't believe this change is warranted. Several others agree as well. Because of this I am going to close this PR. 
/cc @jimangel

**Pull request closed without merging**